src/st/chi/en_salem_witch.c

```diff
@@ -65,6 +67,24 @@ static u8 AnimFrames_TriboltReset[] = {
     0x02, 0x02, 0x02, 0x01, 0x02, 0x02, 0x02, 0x01, 0x03, 0x02, 0x03, 0x01, 0x03, 0x02, 0x04, 0x01,
     0x04, 0x02, 0x05, 0x01, 0x05, 0x02, 0x06, 0x01, 0x06, 0x02, 0xFF, 0x00
 };
+
+// D_8018162C
+static u8 AnimFrames_Twinkle[] = {
+    0x02, 0x29, 0x02, 0x2A, 0x02, 0x2B, 0x02, 0x2C, 0x00, 0x00, 0x00, 0x00};
+
+// D_80181638
+static u8 AnimFrames_CurseProjectile[] = {
+    0x01, 0x2D, 0x01, 0x2E, 0x01, 0x2F, 0x01, 0x30, 0x01, 0x31, 0x01, 0x32, 0x01, 0x33, 0x01, 0x34,
+    0x01, 0x35, 0x01, 0x36, 0x01, 0x37, 0x01, 0x38, 0x01, 0x39, 0x01, 0x3A, 0x01, 0x3B, 0x01, 0x3C,
+    0x00, 0x00, 0x00, 0x00
+};
+
+static u8 AnimFrames_Unused[] = {
+    0x02, 0x01, 0x02, 0x02, 0x02, 0x03, 0x02, 0x04, 0x02, 0x05, 0x02,
+    0x06, 0x02, 0x07, 0x02, 0x08, 0x02, 0x09, 0x02, 0x0A, 0x02, 0x0B,
+    0x02, 0x0C, 0x02, 0x0D, 0x02, 0x0E, 0x00, 0x00, 0x00, 0x00,
+};
+
```

Comment:
Had to move this data up to before `pad2_anim_debug` so the debug string gets put in the right place.